docs/reference/commandline/logs.md

```diff
@@ -19,6 +19,7 @@ weight=1
       --since=""                Show logs since timestamp
       -t, --timestamps=false    Show timestamps
       --tail="all"              Number of lines to show from the end of the logs
+      --log-opt=""              Specify options to the logger, Currently supported only by json-file log driver
```

Comment:
The entire command has a note that says it only works with the json-file log driver --- so you don't need to repeat that on the flag.

"Specify logger options. "  is tighter